test/browser/spec/ol/source/ImageTile.test.js

```diff
@@ -157,6 +157,29 @@ describe('ol/source/ImageTile', () => {
 
       tile.load();
     });
+
+    it('can be set to no-cors', (done) => {
+      const assert = getAssert(done);
+
+      let got;
+      const source = new ImageTile({
+        // TODO: Using `null` to indicate crossOrigin should be unset
+        crossOrigin: null,
+        loader: (x, y, z, {crossOrigin}) => {
+          got = crossOrigin;
+          return loadImage(emptyUrl);
+        },
+      });
+
+      const tile = source.getTile(3, 2, 1);
+      source.on('tileloadend', () => {
+        // TODO: Expecting `null` to indicate crossOrigin was not set
+        assert(got === null, `expected no-cors, got ${got}`);
```

Comment:
```suggestion
        expect(got).to.be(null);
```
Please, at least use the conventions that our other tests use.

The value we got is `'anonymous'`, which is correct according to the API docs. Why are you expecting `no-cors`? That's not a valid value for `Image`'s `crossOrigin` attribute. See https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/crossOrigin.